internal/provider/resource/project_user_resource.go

```diff
@@ -251,6 +251,14 @@ func (r *ProjectUserResource) Create(ctx context.Context, req resource.CreateReq
 		return
 	}
 
+	if len(invitedUser) == 0 {
+		resp.Diagnostics.AddError(
+			"Error inviting user",
+			"Couldn't create project user to Infiscial, no invite was created. Is the user already in the project?",
```

Comment:
Would it make more sense to say: Could not add user to project. No invite was sent. Is the user already in the project?